packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -536,6 +536,11 @@ const errors = {
     level: Level.ERROR,
     docsUrl: `https://www.gatsbyjs.org/docs/gatsby-cli/#new`,
   },
+  // Watchdog
+  "11701": {
+    text: (context): string => context.sourceMessage,
```

Comment:
Please move the error message to this file and pass the variables via `context` to this `text`, like the messages above

Reply:
Updated